src/v/debug_bundle/BUILD

```diff
@@ -137,3 +137,21 @@ redpanda_cc_library(
         "//src/v/bytes",
     ],
 )
+
+redpanda_cc_library(
+    name = "probe",
```

Comment:
The libraries in this subdirectory seem to be extremely fine-grained, is that the recommended way?

Reply:
I could probably reorganize this to have a single 'internal' library rather than multiple ones, but I feel like this is ok?  Any concerns?